stream_alert/athena_partition_refresh/main.py

```diff
@@ -0,0 +1,206 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+import json
+import logging
+import os
+
+from datetime import datetime
+
+import backoff
+import boto3
+
+logging.basicConfig()
+LEVEL = os.environ.get('LOGGER_LEVEL', 'INFO')
+LOGGER = logging.getLogger('StreamAlertAthena')
+LOGGER.setLevel(LEVEL.upper())
+
+ATHENA_CLIENT = None
+DATABASE_DEFAULT = 'default'
+
+
+def _load_config():
+    """Load the Athena Lambda configuration file
+
+    Returns:
+        [dict] Configuration settings by file, includes two keys:
+            lambda: All lambda function settings
+            global: StreamAlert global settings
+    """
+    config_files = ('lambda', 'global')
```

Comment:
`lambda_settings` and `global_settings` - more accurately describes what the key's point to